api/pom.xml

```diff
@@ -26,8 +26,8 @@
         <version>1.0.4</version>
     </parent>
 
-    <groupId>jakarta.annotation</groupId>
     <artifactId>jakarta.annotation-api</artifactId>
+    <groupId>jakarta.annotation</groupId>
```

Comment:
Why did you move the groupId after the artifactId?

Reply:
I did this so that all poms in the project were the same. I don't like when POMs get all gnarly and unorganized. They should be consistent throughout the entire project.

Reply:
How about [POM Code Convention](http://maven.apache.org/developers/conventions/code.html#POM_Code_Convention): _specific POM convention to ordering POM elements_?

Reply:
It's not specific ordering that I'm mostly concerned with, but rather that there is _some form_ of order. I'd be OK with following the convention.